source/includes/v2/_odsql.md

```diff
@@ -134,17 +134,66 @@ Supported geometry expressions are:
-`second`|date field literal|Returns the second of a date|
-`date_format`|date field literal|Returns a formatted date|
+Function|Parameters|Description
+--------|-----------|----------
+`length`|string literal or string field literal|Returns the number of characters
+`now`|optional named parameters (see below)|Returns the current date
+`year`|date field literal|Returns the year of a date
+`month`|date field literal|Returns the month of a date
+`day`|date field literal|Returns the day of a date
+`hour`|date field literal|Returns the hour of a date
+`minute`|date field literal|Returns the minute of a date
+`second`|date field literal|Returns the second of a date
+`date_format`|date field literal|Returns a formatted date
+
+#### Parameters to the now() function
+
+> Examples, assuming the current date time is 2021-05-06 12:34:55.450500+00:00, which is a thursday
+
+```sql
+now() -- Returns '2021-05-06T12:34:55.450500+00:00'
+now(year=2000) -- Sets the year component to return '2000-05-06T12:34:55.450500+00:00'
+now(year=2001, months=-1) -- Sets the year to 2001 and substract 1 month to return '2000-04-06T12:34:55.450500+00:00'
+now(day=31,month=2) -- Sets the day to 31, then the month to 2. The actual day part is rounded to 28 '2021-02-28T12:34:55.450500+00:00'
+now(weekday=0) -- Sets the day to the next monday which is '2021-05-10T12:34:55.450500+00:00'
+now(mondays=+1) -- Sets the day to the next monday which is also '2021-05-10T12:34:55.450500+00:00'
+now(mondays=-1) -- Sets the day to the previous monday which is '2021-05-03T12:34:55.450500+00:00'
+```
+
+Without any parameters, the `now()` function returns the current date and time.
+
+The function may also be called with named parameters to set or modify certain parts of the current date and time.
+
+With each parameter an integer value is required, interpreted as an absolute value or as a relative value to a part of the current date and time.
+
+Parameter names in their singular form will set a certain part of the current date and time to the given value. Parameter names written in plural will add or substract the given value to a part of the current date and time.
+
+If a parameter is used multiple times in the call, only the last one is actually used, the others are ignored.
+
+Parameter name|Accepted values|Description
+--------------|---------------|-----------
+`year`|1 to 9999|Year component
+`years`|Any integer|Value to add to or substract from the year component
+`month`|1 to 12|Month component
+`months`|Any integer|Value to add to or substract from the month component, then the year component in case of overflow
+`day`|Any positive integer|Day component, rounded to the maximum valid day number for the current month
+`days`|Any integer|Value to add to or substract from the day component, then the month component in case of overflow
+`hour`|0 to 23|Hour component
+`hours`|Any integer|Value to add to or substract from the hour component, then the day component in case of overflow
```

Comment:
There's a typo here: "substract" --> subtract.
```suggestion
`hours`|Any integer|Value to add to or subtract from the hour component, then the day component in case of overflow
```